blocks/components/editable/tinymce.js

```diff
@@ -3,7 +3,11 @@ export default class TinyMCE extends wp.element.Component {
 		this.initialize();
 	}
 
-	shouldComponentUpdate() {
+	shouldComponentUpdate( nextProps ) {
+		if ( this.editorNode.getAttribute( 'data-is-empty' ) !== nextProps.isEmpty ) {
```

Comment:
>I'm not sure if I get this. It doesn't wrap the TinyMCE component?

We can change it so that it does 😄  Or more accurately, with #671, we _are_ changing it so that it does.